### ISSUE TITLE
Species synonyms for the genus Nyctophilus in ITIS

### ISSUE THREAD

**Issue by mdoering** [CatalogueOfLife/data#812]
@DaveNicolson How can the genus [Nyctophilus Leach, 1821](https://www.itis.gov/servlet/SingleRpt/RefRpt?search_topic=Scientific_Author&search_value=Leach%2C+1821) have 3 species as synonyms in ITIS?

https://www.itis.gov/servlet/SingleRpt/SingleRpt?search_topic=TSN&search_value=631385#null

**Comment by yroskov** [comment]
ITIS approach has a sense to me. 

However, if the GSD flags the binomial names Vespertilio timoriensis & Nyctophilus timoriensis as _nomen dubium_ and does not point them to an accepted parent, the CoL should recognize them as "bare names".

**Comment by mdoering** [comment]
We need some best practice for dealing with invalid names and doubtful names.
We do have rules for dealing with unplaced, but accepted species, which can be linked to a higher taxon instead of their genus. 

When it is unclear or even known that a name is not valid, we often still want to place it somewhere so it can be found in taxonomic context. It is of interest to know these are (potential) bat names. Bare names cannot be placed anywhere, nomenclature does not have a classification. This is problematic. But even IPNI and other nomenclators put names into a family box to organise them. It might be best to create a new "taxonomic" status `doubtful`, `uncertain` or `unresolved` that still allows to classify the names, but clearly tells these names are not to be used. [WFO](https://about.worldfloraonline.org/images/uploads/documents/WFOGuidelinesForTaxonomicBackboneContributorsV._2.06.pdf) for examples uses `unchecked` (A name that has not been checked by the contributing TEN) and `ambiguous` (A name that has been checked by the contributing TEN but could not be resolved as accepted or synonym). See also suggestion to add [unplaced](https://github.com/CatalogueOfLife/coldp/issues/84) in ColDP.

**Comment by mdoering** [comment]
> As one of our respected author said, I spent my entire professional life cleaning my checklist of the garbage of floating names that have no relation to taxonomic concepts, and I don't want anyone to put them back.

Agree, but if you never know which these names are and have them marked as "garbage" they will always creep back. We need to know about them and have a way to deal with them.
